### ISSUE TITLE
Spin  button of column count in table block is not access by keyboard user.

### ISSUE THREAD

**Issue by nsingh2001** [nsingh2001/Addpage-WP#1]
UAT 2022-09
Windows 10
Chrome Version 100.0.4896.88 (Official Build) (64-bit)
NVDA-2022

Repro step
1.	open URLhttps://test-wp-manish.azurewebsites.net/wp-admin/ 
2.	go to Page link and then Add new link.
3.	Tab to block inserter and insert table block.
4.	Go to table block and then use down arrow to reach column count spin button.
Expected Result:
The spin button of column count should be accessible by keyboard user.

Actual Result :
The value of column count is not change by up and down arrow  of keyboard.

Applicable to WCAG success criteria  :
https://www.w3.org/TR/WCAG21/#keyboard

Attached 
[Spin button.webm](https://user-images.githubusercontent.com/102017896/191167371-b678916f-9d7a-4017-964c-0214a235345e.webm)
file
